### DISCUSSION TITLE
encore.cluster is very slow.

### DISCUSSION THREAD

**Discussion by khoroshyy** [MDAnalysis/mdanalysis#4789] [Q&A]
Hi, I have a trajectory of a ligand and want to cluster conformations (70 atoms) 10000  frames.
The trajectory is centered and aligned.
I try to cluster it using this command 
and cluster_collection = encore.cluster(u, select="resnum 10834 and not name H*", ncores=10)
But it takes forever; any hints on how to speed it up or at least see the progress?
Thanks.
Petro.

**Comment by orbeckst** [comment]
I am not an expert on ENCORE so I am not sure how long it should take. If I were to try it, I'd first use it on a small system where I know the answer and learn how it behaves.

Development on ENCORE has slowed down and it's now its own MDAKit. If you have a suggestion for improvement such as at least adding a progress bar (which sounds like a great idea to me) then please raise it at https://github.com/MDAnalysis/mdaencore/issues .

**Comment by khoroshyy** [comment]
Thanks
